app/ante/fees.go

```diff
@@ -124,15 +121,7 @@ func CheckEthMinGasPrice(tx sdk.Tx, minGasPrice sdkmath.LegacyDec, baseFee *big.
 		// increase the GasTipCap (priority fee) until EffectivePrice > MinGasPrices.
 		// Transactions with MinGasPrices * gasUsed < tx fees < EffectiveFee are rejected
 		// by the feemarket AnteHandle
-
-		txData, err := evmtypes.UnpackTxData(ethMsg.Data)
-		if err != nil {
-			return errorsmod.Wrapf(err, "failed to unpack tx data %s", ethMsg.Hash)
-		}
-
-		if txData.TxType() != ethtypes.LegacyTxType {
```

Comment:
do we keep this TxType check

Reply:
The single call covers both case I think